src/Symfony/Bridge/Doctrine/DependencyInjection/CompilerPass/RegisterMappingsPass.php

```diff
@@ -178,7 +178,7 @@ protected function getChainDriverServiceName(ContainerBuilder $container)
     /**
      * Create the service definition for the metadata driver.
      *
-     * @param ContainerBuilder $container passed on in case an extending class
+     * @param ContainerBuilder $container Passed on in case an extending class
```

Comment:
such changes should probably be reverted as https://github.com/symfony/symfony/pull/24149 added a cs rule against doing sentences in phpdoc (no uppercase, no ending dot)

Reply:
I don't understand the reasoning given there.
In fact, having an uppercase letter makes the docblock easier to read as the eyes more obviously can spot when a new column starts.
The code base is also using uppercase everywhere else. This is not by chance, but because it is most natural to others.
In short, this is something I did on purpose.

Reply:
I don't understand the reasoning given there.
In fact, having an uppercase letter makes the docblock easier to read as the eyes more obviously can spot when a new column starts.
The code base is also using uppercase everywhere else. This is not by chance, but because it is most natural to others.
In short, this is something I did on purpose.

Reply:
And I fully agree, we just did the contrary when merging it.